docs/ops/deployment/kubernetes.md

```diff
@@ -112,20 +153,38 @@ spec:
       containers:
       - name: jobmanager
         image: flink:latest
-        args:
-        - jobmanager
+        workingDir: /opt/flink
+        command: ["/bin/bash", "-c", "$FLINK_HOME/bin/jobmanager.sh start;\
+          while :;
+          do
+            if [[ -f $(find log -name '*jobmanager*.log' -print -quit) ]];
+              then tail -f -n +1 log/*jobmanager*.log;
+            fi;
+          done"]
         ports:
         - containerPort: 6123
           name: rpc
         - containerPort: 6124
           name: blob
-        - containerPort: 6125
-          name: query
         - containerPort: 8081
           name: ui
-        env:
-        - name: JOB_MANAGER_RPC_ADDRESS
-          value: flink-jobmanager
+        livenessProbe:
+          tcpSocket:
+            port: 6123
+          initialDelaySeconds: 30
+          periodSeconds: 60
+        volumeMounts:
+        - name: flink-config-volume
+          mountPath: /opt/flink/conf
+      volumes:
+      - name: flink-config-volume
```

Comment:
Indentation seems to be wrong

Reply:
Actually, this is the right indentation. `containers` and `volumes` are both fields in [podspec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#podspec-v1-core).

Reply:
Actually, this is the right indentation. `containers` and `volumes` are both fields in [podspec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#podspec-v1-core).